docs/contributing.rst

```diff
@@ -62,6 +62,10 @@ We welcome your patronage on `Bountysource <https://www.bountysource.com/teams/u
 Your contribution will go towards adding new features to urllib3 and making
 sure all functionality continues to meet our high quality standards.
 
+We also welcome sponsorship in the form of time; by allowing your engineers to
+spend time improving urllib3 and contributing those improvements back to the
```

Comment:
Maybe emphasize ongoing/consistent time. I'd want to differentiate between allowing X hours/wk to work on urllib3 vs a single PR (which is appreciated but not the same thing as sponsorship IMO).

Reply:
@shazow, thoughts on the new wording?

Reply:
It's probably fine, but I still imagine some company being like "well we sent a basic PR so we're a sponsor now." You're the lead, so your call. :D 

I imagine something like:

> We also welcome sponsorship in the form of development time, such as by encouraging employees to contribute to urllib3 in a recurring way during their work hours.

Reply:
That's fair. Lemme cogitate a bit.

Reply:
@shazow, one more look. 😉